share/instruments/instruments.xml

```diff
@@ -9464,6 +9496,35 @@
                   <genre>concertband</genre>
                   <genre>classroom</genre>
             </Instrument>
+            <Instrument id="cannon">
+                  <family>other-percussion</family>
+                  <trackName>Cannon</trackName>
+                  <longName>Cannon</longName>
+                  <shortName>Cn.</shortName>
+                  <description>Military cannon or gunshot sound.</description>
```

Comment:
I added "military" to the start of your description to make it clear that this is (supposed to be) a real cannon rather than some kind of musical instrument that happens to be called a "cannon".